synapse/types/__init__.py

```diff
@@ -979,3 +980,40 @@ class UserProfile(TypedDict):
 class RetentionPolicy:
     min_lifetime: Optional[int] = None
     max_lifetime: Optional[int] = None
+
+
+class TaskStatus(str, Enum):
```

Comment:
I think we usually put classes like this right in the store file, but it doesn't matter too much.

Reply:
I think I prefer keeping it here because it feels a bit wrong to have them in the DB package since they are leaking outside of the `TaskScheduler` util class.